server/service/FlightService.js

```diff
@@ -452,6 +453,69 @@ const flightService = {
     await storeFixesToDB(flight, fixes, fixesStats);
   },
 
+  /**
```

Comment:
Puh... der FlightService hat jetzt schon mehr als 1000 Zeilen. Das würde ich lieber in ein eigenes File ausglidern.
Können wir aber auch gerne später machen.
Dann können wir vielleicht ein flight-utils Ordner machen, in dem die anderen Files aus /igc auch rein kommen